scipy/stats/_multivariate.py

```diff
@@ -448,21 +248,20 @@ def random_state(self, seed):
 
 
 _mvn_doc_default_callparams = """\
-mean : array_like, optional
-    Mean of the distribution (default: origin)
-cov : array_like, optional
-    Covariance matrix of the distribution (default: identity)
-allow_singular : bool, optional
-    Whether to allow a singular covariance matrix.  (default: False)
+mean : array_like, default: ``[0]``
+    Mean of the distribution.
+cov : array_like, default: ``[1]``
```

Comment:
We need to update `multivariate_normal` documentation to reflect that we can now pass `CovViaPrecision`:
```suggestion
cov : array_like or Covariance, default: ``[1]``
```